data/strings/strings.txt

```diff
@@ -18750,42 +18750,36 @@
 
   [navigation_stop_button]
     tags = android,ios
-    en = stop
-    af = stop
```

Comment:
Was it lost?

Reply:
Not my PR, but a doubt I have: if the language string is equal to English, shall it be duplicated or not? Since EN is used as a fallback, perhaps this makes the strings file smaller?

Reply:
@matheusgomesms you're right! I forgot about it 🤦 
Checking generated strings confirms that strings are generated with default En for iOS (otherwise it doesn't work, we need to patch it to fall back to en) and are NOT generated for Android (it properly reuses en values).

Reply:
yup this was why :)